web/styles/dark_theme.css

```diff
@@ -791,14 +791,18 @@
     #settings_page .settings-header,
     #settings_page .sidebar li.active,
     #settings_page .sidebar-wrapper .tab-container,
-    .table-striped tbody tr:nth-child(even) td,
     .table-striped tbody tr:nth-child(odd) th,
     #subscription_overlay #stream-creation .settings-sticky-footer,
     #groups_overlay #user-group-creation .settings-sticky-footer {
         border-color: hsl(0deg 0% 0% / 20%);
         background-color: hsl(0deg 0% 0% / 20%);
     }
 
+    .table-striped tbody tr:nth-child(even) td {
+        border-color: hsl(0deg 0% 0% / 20%);
+        background-color: hsl(212deg 28% 15%);
```

Comment:
I am not sure how you arrived at this color, but this does not seem to be the same as the one that we use currently. There has been some discussion here about how to update the color to not use `alpha` value in https://chat.zulip.org/#narrow/stream/6-frontend/topic/alphas.20in.20color.20definitions.

Reply:
so would it be fine if i update color value by using color-mix?